### DISCUSSION TITLE
Ring on Button causes weird glitch

### DISCUSSION THREAD

**Discussion by DanielK1511** [tailwindlabs/tailwindcss#11355] [Help]
![tailwindi](https://github.com/tailwindlabs/tailwindcss/assets/36012642/45f21fd9-f91f-4677-bd0c-0deaa2f21372)

Hey Guys i am currently fiddling around with rings that appear on focus.
It is working but its causing a weird glitch where the buttons corners appear to have the corners colored in the same color of the ring.
(Look closely in the image)
I have used 2 seperate PCs and different browsers to verify, but i dont see where the issue is thats causing this.

My Code:
```html
<a href="#" class="bg-Grau-0 block focus:ring-2 focus:ring-gruen-0 focus:ring-offset-4 focus:ring-offset-GrauHintergrund-0 font-semibold hover:bg-border2-0 mt-10 px-3 py-3 rounded-2xl text-center text-sm text-white w-full">Jetzt abonnieren</a>
```

Maybe you can tell me what i am doing wrong. 
Thanks :)

**Comment by hibatillah** [comment]
is this color `focus:ring-offset-GrauHintergrund-0` ? i think so

so i think first you add `ring-gruen-0 ` for ring and you make it offset right, then you give color for `ring-offset-GrauHintergrund-0`
it could be that the **ring color** overlaps with the **ring offset color**, and the glitch that u mean is ***ring color***

just my opinion, cmiiw guys, i'm still learning:)
